lib/driver/pg.js

```diff
@@ -290,8 +290,18 @@ var PgDriver = Base.extend({
         } else {
           sql = util.format('ALTER TABLE "%s" ALTER COLUMN "%s" DROP DEFAULT', tableName, columnName);
         }
+        return this.runSql(sql).then(
+          setType.bind(this)
```

Comment:
Oh my fault, may be I was not completely awake this morning. No everything is fine just ignore what I have written, I have seen a closure where no closure was.
Sorry for that.